pkg/controller/nodedaemon/reconcile.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	// ProvisionerName is the name of the local-static-provisioner daemonset
-	ProvisionerName = "localvolumeset-local-provisioner"
+	ProvisionerName = "local-provisioner"
```

Comment:
This value is used in the `cleanupOldDaemonsets()` to cleanup older deamonsets, can you preserve this as `oldProvisionerName` (unexported) and change the usages in that function to that value?